### ISSUE TITLE
Menu names are unreadable on Ubuntu 18.10

### ISSUE THREAD

**Issue by jayfoad** [Dyalog/ride#511]
### Describe the issue you are having
On Ubuntu 18.10 the menu names (File, Edit etc) seem to be in grey on grey and are almost impossible to read:

![screenshot from 2019-01-11 10-29-48](https://user-images.githubusercontent.com/108316/51028763-4fcdc180-158c-11e9-8d6d-9cd7450af081.png)

### Paste the contents of Help → About (Shift+F1)
```
IDE:
  Version: 4.2.3413
  Platform: Linux x86_64
  Date: 2019-01-07 13:31:17 +0000
  Git commit: 81d6ae270e316dcfdf666a157c1e8709c4b4cabd
  Preferences:{

  }

Interpreter:
  Version: 18.0.35177
  Platform: Linux-64
  Edition: Unicode/64
  Date: Jan 10 2019 at 16:49:46
```

**Comment by jayfoad** [comment]
It's an Electron issue: electron/electron#15194

**Comment by jayfoad** [comment]
The fix is in Electron 3.0.14, 3.1.1 and 4.0.1. @e9gille can we upgrade?

**Comment by e9gille** [comment]
Sure, working on upgrading to 4.0 on the master branch.

**Comment by e9gille** [comment]
fixed in 5faae02ee30b181f2814c2181d3415167dc53191
